draft-ietf-rats-corim.md

```diff
@@ -2174,20 +2174,18 @@ See {{sec-ir-cm}}.
 Verifiers are not required to use this as their internal representation.
 For the purposes of this document, appraisal is described in terms of the above cited internal representation.
 
-[^issue] https://github.com/ietf-rats-wg/draft-ietf-rats-corim/issues/232
-
 #### ACS Processing Requirements
 
 The ACS contains the actual state of Attester's Target Environments (TEs).
-The `state-triples` field contains Evidence (from Attesters) and Endorsements
+The ACS contains Evidence ECTs (from Attesters) and Endorsements ECTs
 (e.g. from `endorsed-triple-record`).
 
 CoMID Reference Values will be matched against the ACS following the comparison rules in {{sec-match-condition-ect}}.
 This document describes an example evidence structure which can be
 matched against these Reference Values.
 
-Each entry within `state-triples` uses the syntax of `endorsed-triple-record`.
-When an `endorsed-triple-record` appears within `state-triples` it
+Each entry within Endorsements ECTs uses the syntax of `endorsed-triple-record`.
```

Comment:
```suggestion
Each Endorsement ECT contains the environment and internal representation of `measurement-map`s as extracted from an `endorsed-triple-record`.
```
What is an entry in an ECT? I understand an entry as something in the attestation context, like ACS, ae, rv, etc.. This also isn't exactly using the same syntax for environment and element-list as an `endorsed-triple-record`, since we use internal representations.

Reply:
i am ok with this clarity, works for me!